pkg/project/project.go

```diff
@@ -608,14 +610,23 @@ func (e *Event) Name() string { return e.event.Name }
 
 // Value returned with all secrets evaluated
 func (e *Event) Value() (string, error) {
+	if e.BashifyPaths {
+		return ExpandFromProjectBashifyPaths(e.event.Value, e.project)
+	}
```

Comment:
This would lead to us bashifying values that aren't paths. I don't think we need to be expanding this value for bash paths for bash compatibility to work?

Reply:
This only provides a context such that if a `.path()` is encountered, it is bashified. It will not bashify just any value.

Reply:
I see. I think this would be better addressed at the source:

https://github.com/ActiveState/cli/blob/4b38cc68f36736428be2ad5e42a94c9e431b414f/pkg/project/expander.go#L274-L274

Your solution at the moment appears to be trying to update the path once it has already been evaluated, I think?

Reply:
No. This is when there is a path call inside the script itself. For example, see the changes at the top of this PR, and also review the activestate.yaml.*.tpl templates, which have `.path()` calls in them.

Reply:
As far as I'm aware the code I linked is what resolves the path call inside the script itself.